src/cpu/aarch64/jit_uni_dw_conv_kernel_f32.cpp

```diff
@@ -1097,11 +1097,14 @@ jit_uni_dw_conv_bwd_weights_kernel_f32<isa>::compute_ow_block_unroll() {
 template <cpu_isa_t isa>
 void jit_uni_dw_conv_bwd_weights_kernel_f32<isa>::generate() {
     preamble();
-    if (simd_w == 8) {
+
+    if (simd_w == 16)
+        ptrue(P_ALL_ONE.b);
+    else if (simd_w == 8)
         ptrue(P_ALL_ONE.b, VL32);
-    } else {
+    else
         assert(!"Unsupport: simd_w != 16, 8");
-    }
```

Comment:
Is this expected that this PR includes previous PR about dw convolution, but merges this hunk somehow to the 128 and 256 SVE commit?

Reply:
@mgouicem 
Thank you.
https://github.com/oneapi-src/oneDNN/pull/1383 and https://github.com/oneapi-src/oneDNN/pull/1384 had duplicate commits. I removed them from https://github.com/oneapi-src/oneDNN/pull/1384

Reply:
Thanks!